src/main/java/seedu/address/model/localcourse/LocalCourse.java

```diff
@@ -73,7 +79,8 @@ public boolean equals(Object other) {
 
         LocalCourse otherLocalCourse = (LocalCourse) other;
         return localCode.equals(otherLocalCourse.localCode)
-                && localName.equals(otherLocalCourse.localName);
+                && localName.equals(otherLocalCourse.localName)
+                && localUnit.equals(otherLocalCourse.localUnit);
     }
 
     @Override
```

Comment:
Is there any specific reason why LocalUnit is not included in this method?

Reply:
localunit has been included, not too sure which you are refering too..